recipe/meta.yaml

```diff
@@ -94,22 +103,22 @@ requirements:
     - fftw * mpi_{{ mpi }}_*
     - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
     - cudatoolkit                               # [(cuda_compiler_version or "").startswith("11")]
-    {% if cuda_major >= 12 %}
+{% if cuda_major >= 12 %}
```

Comment:
The currently used parser doesn't handle jinja conditions when they are indented. This is a bug: https://github.com/conda-incubator/conda-recipe-manager/issues/258